spectral/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "npm run clean && tsc && npm run copy-ruleset",
     "clean": "rm -rf dist",
     "lint": "eslint src/**/*.ts",
-    "copy-ruleset": "cp ./ruleset.yaml ./dist/ruleset.yaml && cp ./package.json ./dist/package.json",
+    "copy-ruleset": "cp rhoasapi ruleset.yaml package.json ./dist ",
```

Comment:
copying the binary here?

Reply:
yes. it needs to be in the dist

Reply:
ok, I did that before and you didn't like it?